modules/customizing-the-appearance/proc-customize-rhdh-sidebar-menuitems.adoc

```diff
@@ -1,129 +1,60 @@
-[source,html]
-----
-icon: <svg width="20px" height="20px" viewBox="0 0 512 512" xmlns="http://www.w3.org/2000/svg" fill="#ffffff">...</svg>
-----
-+
-.Example image icon
-[source,html]
-----
-icon: https://img.icons8.com/ios-glyphs/20/FFFFFF/shop.png
-----
-
-<3> This optional attribute specifies the title of the menu item. It can be removed if the title is already specified in the `dynamicRoutes` configuration under `menuItem.text`.
-
-<4> This optional attribute sets the order in which menu items appear in the sidebar. The default priority is 0, which places the item at the bottom of the list. A higher priority value places the item higher in the sidebar. You can define this attribute for each section.
-
-<5> This optional attribute specifies the parent menu item under which the current item is nested. If this attribute is used, the parent menu item must be defined elsewhere in the `menuItems` configuration of any enabled plugin. You can define this attribute for each section.
-
-.Example `menuItems` configuration
 [source,yaml]
 ----
 dynamicPlugins:
   frontend:
-    <package_name>:
-      dynamicRoutes:
-        - path: /my-plugin
-          module: CustomModule
-          importName: FooPluginPage
-          menuItem:
-            icon: fooIcon
-            text: Foo Plugin Page
+    default.main-menu-items: # <1>
       menuItems:
-        my-plugin: # matches `path` in `dynamicRoutes`
-          priority: 10 # controls order of plugins under the parent menu item
-          parent: favorites # nests this plugin under the `favorites` parent menu item
-        favorites: # configuration for the parent menu item
-          icon: favorite # icon from RHDH system icons
-          title: Favorites # title for the parent menu item
-          priority: 100 # controls the order of this top-level menu item
-----
---
-
-. To ensure that a menu item is identified as a main menu item, you must add the `default.` prefix to its key. For example:
+        default._<menu_item_name>_: # <2>
+          parent: _<my_menu_group>_ # <3>
+          priority: 10 <4>
+        default._<menu_group_parent_item_name>_: # <4>
+          icon: _<my_menu_group_icon>_ # <5>
+          title: _<my_menu_group_title>_ # <6>
+          priority: 100 # <7>
+----
+<1> `default.main-menu-items`: Specifies the key for configuring static main menu items. The `default.` prefix identifies an item as a main menu item. You can add the `default.` prefix to both individual menu items or parent menu group configuration, such as `default.<menu_group_parent_item_name>` in the previous example.
+<2> `default._<menu_item_name>_`: Specifies the key of the menu item configuration. The `default.` prefix is required for a main menu item key.
+<3> `parent`: (Optional) Specifies the parent menu item for this item.
+<4> `priority`: (Optional) Specifies the order of this menu item within its menu level.
+<5> `icon`: Specifies the icon for the menu group. Required for parent menu items.
```

Comment:
A description for the `default._<menu_group_parent_item_name>_:` callout is missing here.